### PR TITLE
fix(preview): Changed table justifyContent to start

### PR DESCRIPTION
**Pull request by vibdev**
## Summary

Fixes: #2555 

Fixes the preview table not flipping header cells correctly in rtl mode

## Release Category
Components

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [Documentation Guidelines](https://workday.github.io/canvas-kit/?path=/docs/guides-documentation-guidelines--page)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Testing Manually

Test by flipping storybook to rtl

## Screenshots or GIFs

Before
Screenshot 2024-02-06 at 5 59 39 PM

After
Screenshot 2024-02-06 at 5 59 26 PM

## Thank You Gif (manditory)

missy elliott flipping and reversing it

### CODE REVIEW

**Comment by mannycarrera4** [comment]
Any chance you have time to add a RTL storybook example? I'm surprised we didn't have one

**Comment by vibdev** [comment]
@manny https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOXozdnJsb2xkcW5xNzN0MjZyajF5Z3hzanh1NWFybTB0NzJhcDJ0cSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/HjPbLbmep2aJO/giphy.gif

File: modules/preview-react/table/lib/TableCell.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const TableCell = createComponent('td')({
         backgroundColor="frenchVanilla100"
         borderBottom={`1px solid ${colors.soap400}`}
         gridTemplateColumns="1fr"
-        justifyContent="left"
+        justifyContent="flex-start"
```
Comment (alanbsmith) [inline modules/preview-react/table/lib/TableCell.tsx:17]: Nit: This works, but the component is `display: grid`, not `flex`.
```suggestion
        justifyContent="start"
```

File: modules/preview-react/table/lib/TableHeader.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const TableHeader = createComponent('th')({
         alignItems="center"
         backgroundColor="frenchVanilla100"
         borderBottom={`1px solid ${colors.soap400}`}
-        justifyContent="left"
+        justifyContent="flex-start"
```
Comment (alanbsmith) [inline modules/preview-react/table/lib/TableHeader.tsx:16]:
```suggestion
        justifyContent="start"
```

**Review by alanbsmith — APPROVED** [review]
Looks good. A couple (admittedly very nit) tweaks. Appreciate the fix, @vibdev!

**Review by mannycarrera4 — APPROVED** [review]

**Review by josh-bagwell — APPROVED** [review]

**Pull request merged**